.dockerignore

```diff
@@ -0,0 +1,6 @@
+build/**/*
```

Comment:
wouldn't just `build/` do the same job here?

Reply:
probably yes, still confused between the .gitignore syntax and this one

Reply:
oh wait - `.dockerignore` - sorry, I mistook it as well. could be correct for all I know :P

Reply:
in the end it was correct to use `build/` as the other form is only necessary to exlude, say, all `*.php` files from any of the subdirectory (`build/**/*.php`) but not other files
in `.gitignore` this would be `/build/